media-video/mpv/mpv-9999.ebuild

```diff
@@ -44,7 +44,6 @@ REQUIRED_USE="
 "
 RESTRICT="!test? ( test )"
 
-# raspberry-pi: default to -bin given non-bin is known broken (bug #893422)
```

Comment:
Can do `Closes: https://bugs.gentoo.org/893422`, glad won't have to worry about that messy package anymore.

Beside that, lgtm for mpv. Don't have a rpi so I'll trust whatever you feel works out better for it, thanks.

I'll keep myself a note to cleanup the mpv rpi profile stuff after stable+cleanup.

Reply:
On a side-note, mpv-0.37.0 has new defaults that may be hard on a rpi compared to 0.36.0

If have speed issues, try --profile=fast to revert back to mpv-0.36.0 defaults.

Reply:
I didn't realise that changed in 0.37.0. I am aware of `--profile=fast`, which I gather is basically the same as `--scaling=bilinear`. That's fine, but it still might be nice to be able to choose rendering directly to the overlay.